tests/DefaultFieldResolverTest.php

```diff
@@ -34,12 +34,14 @@ public function getEntity()
             ['getPublic', new IgnoredGetter(), 'public'],
             [['real entity', 2, ['foo']], new IgnoredGetter(), 'publicWithArgs', ['arg2' => 2, 'arg1' => $entityID]],
             [null, new IgnoredGetter(), 'nonExisting'],
-            [null, new IgnoredGetter(), '__call'],
+            ['__call', new IgnoredGetter(), '__call', ['name' => 'unittest', 'arguments' => 'none']],
             [true, new IgnoredGetter(), 'isValid'],
             [true, new IgnoredGetter(), 'hasMoney'],
             ['john', new DefaultValue(), 'nameWithDefaultValueOnArgument'],
             ['jane', new DefaultValue(), 'nameWithDefaultValueOnArgument', ['name' => 'jane']],
             ['bar', ['foo' => 'bar'], 'foo'],
+            [true, new IgnoredGetter(), 'isser'], // test is method prefix
+            [true, new IgnoredGetter(), 'money'], // test has method prefix
```

Comment:
Those cases should not be valid. For the API user, the field `"money"` lose semantic compare to the original `"hasMoney"`

Reply:
If so then the above "is${inflection}", "has${inflection}" should be removed in favor of just a getter /guess/.